versioned_docs/version-8.4/reference/announcements.md

```diff
@@ -38,6 +38,27 @@ As of the 8.4 release, Zeebe, Operate, and Tasklist are now compatible with [Ama
 The Helm charts are not yet prepared with the OpenSearch configurations as templates/pre-filled. The Helm charts can still be used to install for OpenSearch, but some adjustments are needed beforehand. Refer to the [Helm deployment documentation](/self-managed/platform-deployment/helm-kubernetes/deploy.md) for further details.
 :::
 
+### Known limitations
+
+This release contains the following limitations:
+
+- In **Operate `8.4.0`**
+  - **Bug**
+    - **Description:** Instance migration always points to the latest process version
+    - **Reference:** https://github.com/camunda/operate/issues/6091
```

Comment:
Our repo is not public, so if this is on announcements I've created a public issue to rather show there: https://github.com/camunda/issues/issues/567

Reply:
Thanks, corrected